Source/Core/VideoBackends/Software/Rasterizer.cpp

```diff
@@ -371,9 +371,9 @@ static void DrawTriangleFrontFace(const OutputVertexData* v0, const OutputVertex
 
   // scissor
   ASSERT(scissor.rect.left >= 0);
-  ASSERT(scissor.rect.right <= EFB_WIDTH);
+  ASSERT(scissor.rect.right <= int(EFB_WIDTH));
```

Comment:
```suggestion
  ASSERT(scissor.rect.right <= static_cast<int>(EFB_WIDTH));
```
... and likewise for the other casts.